app/serializers/disability_claim_base_serializer.rb

```diff
@@ -30,4 +31,8 @@ def open
   def waiver_submitted
     object.data['waiver5103Submitted']
   end
+
+  def development_letter_sent
+    object.data['developmentLetterSent'] == 'Yes'
```

Comment:
We can remove it later, I just wanted to get something out there for the front end team to be able to wire up the list view for the time being